.php-cs-fixer.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+$finder = PhpCsFixer\Finder::create()->in([__DIR__]);
+
+return (new PhpCsFixer\Config())
```

Comment:
if you feel like it: https://github.com/theofidry/php-cs-fixer-config

Or alternatively create your own, it's relatively easy.